docs/src/pages/[platform]/connected-components/account-settings/delete-user/react.mdx

```diff
@@ -1,2 +1,164 @@
-## Delete User
-TODO
+import ReactPropsTable from '@/components/propsTable/ReactPropsTable';
+import { DELETE_USER, OVERRIDES, WARNING } from './props';
+
+### Props
+
+<ReactPropsTable props={DELETE_USER} />
+
+### Basic Usage
+
+`DeleteUser` has `onSuccess` handler that will be called after successful user account deletion. You may use this callback to run any custom handling, such as navigating route or rendering confirmation messages.
+
+```jsx
+import React from 'react';
+import { DeleteUser } from '@aws-amplify/ui-react';
+
+function App() {
+  const handleSuccess = () => {
+    alert('password is successfully changed!')
+  }
+  
+  return (
+    <DeleteUser onSuccess={handleSuccess}/>
+  );
+}
+```
+
+
+### Override default delete handler
+You can override the default deleteUser handler with `handleDelete` prop to add any custom cleanup handlers before or after account deletion.
+
+```jsx
+import React from 'react';
+
+import { Auth } from 'aws-amplify';
+import { DeleteUser } from '@aws-amplify/ui-react';
+
+function App() {
+  const handleSuccess = () => {
+    alert('password is successfully changed!')
+  }
+
+  const handleDelete = async () => {
+    console.log("Doing some clean up...");
+    console.log("Done!");
+    await Auth.deleteUser();
+  };  
+      
+  return (
+    <DeleteUser onSuccess={handleSuccess}/>
+  );
```

Comment:
Does `handleDelete` not need to be passed into the `DeleteUser` component?

Reply:
Thanks for the catch 🤦

```suggestion
You can override the default deleteUser handler with `handleDelete` prop to add any custom cleanup handlers before or after account deletion.

```jsx
import React from 'react';

import { Auth } from 'aws-amplify';
import { DeleteUser } from '@aws-amplify/ui-react';

function App() {
  const handleSuccess = () => {
    alert('password is successfully changed!')
  }

  const handleDelete = async () => {
    console.log("Doing some clean up...");
    console.log("Done!");
    await Auth.deleteUser();
  };  
      
  return (
    <DeleteUser onSuccess={handleSuccess}/ handleDelete={handleDelete}>
  );
```